.changeset/nice-impalas-whisper.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/cloudflare': patch
+---
+
+Fixes regression in Astro asset's image service
```

Comment:
```suggestion
Fixes a regression in `astro:assets` where.... (explain thing that happened here)
```
If there was a regression, it should be possible to say what happened?

But, looking at the error message, it says that it has switched you to the noop image service? This seems like a bigger deal. So is what's really happening here more like:

> Identifies when image optimization is not possible and configures a `noop` image service for the user with a warning

?

Reply:
Perhaps, if I may weigh in here, here's a couple I thought of:

```
1. Fixes accidental overwrite of a valid `astro:assets` image service configuration.
2. `astro:assets` image service now only overrides invalid entry point configurations.
```

Hopefully either of these helps!